internal/dataprovider/dataprovider.go

```diff
@@ -3979,7 +3979,26 @@ func getExternalAuthResponse(username, password, pkey, keyboardInteractive, ip,
 		fmt.Sprintf("SFTPGO_AUTHD_PROTOCOL=%v", protocol),
 		fmt.Sprintf("SFTPGO_AUTHD_TLS_CERT=%v", strings.ReplaceAll(tlsCert, "\n", "\\n")),
 		fmt.Sprintf("SFTPGO_AUTHD_KEYBOARD_INTERACTIVE=%v", keyboardInteractive))
-	return cmd.Output()
+
+	var stdout bytes.Buffer
+	cmd.Stdout = &stdout
+
+	stderr, err := cmd.StderrPipe()
+	if err != nil {
+		return nil, err
+	}
+
+	err = cmd.Start()
+	if err != nil {
+		return nil, err
+	}
+
+	in := bufio.NewScanner(stderr)
+	for in.Scan() {
+		logger.Log(logger.LevelWarn, "external_auth_hook", "", "%s", in.Text())
```

Comment:
This can be simplified as

```
logger.Log(logger.LevelWarn, "external_auth_hook", "", in.Text())
```

and we should probably also avoid logging blank lines otherwise we'll have this

{"level":"warn","time":"2023-02-26T07:28:05.731","sender":"external_auth_hook"}